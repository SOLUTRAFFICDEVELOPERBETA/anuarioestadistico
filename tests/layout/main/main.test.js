import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import Navigator from '../../../Layout/main';

describe('Pruebas en Navigator', () => {
    const children = (<p><b>Testing</b> Pruebas</p>)
    const wrapper = shallow(
        <Navigator
            children={children}
        />

    )

    test('debe mostrarse correctamente', () => {
        expect(wrapper).toMatchSnapshot();
    })

    test('debe de mostrar la informaciÃ³n del children', () => {
        expect(wrapper.find('p').html()).toBe('<p><b>Testing</b> Pruebas</p>')
    })

    test('debe de mostrar los componentes <Header/> y <MainFooter/>', ()=> {
        const header = wrapper.find('Header');
        const footer = wrapper.find('MainFooter');

        expect(header.exists()).toBe(true);
        expect(footer.exists()).toBe(true);
    })
})
