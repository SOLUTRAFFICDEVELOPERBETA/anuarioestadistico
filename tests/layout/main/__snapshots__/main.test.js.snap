// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en Navigator debe mostrarse correctamente 1`] = `
<div>
  <Header />
  <main>
    <div>
      <p>
        <b>
          Testing
        </b>
         Pruebas
      </p>
    </div>
  </main>
  <MainFooter />
</div>
`;
